velox/vector/tests/VectorSaverTest.cpp

```diff
@@ -622,14 +622,6 @@ TEST_F(VectorSaverTest, LazyVector) {
       fuzzer);
 }
 
-TEST_F(VectorSaverTest, stdVector) {
```

Comment:
Why is this unit test removed? Should we cover it in FuzzerToolkitTest?

Reply:
Would it worth creating a unit test for InputRowMetadata::saveToFile() and InputRowMetadata::restoreFromFile() instead?